api/cpp/CMakeLists.txt

```diff
@@ -398,3 +398,15 @@ include(CPack)
 if(SLINT_BUILD_TESTING)
     add_subdirectory(tests)
 endif()
+
+if(SLINT_BUILD_EXAMPLES)
```

Comment:
Ah, you went for `api/cpp/examples` instead of `examples/cpp`. Fine with me, although I think everything under `examples/` is a little more discoverable (since it's closer to the root), for somebody browsing the repo, while looking under `api/cpp` is two levels deeper.